libs/state/package.json

```diff
@@ -6,11 +6,6 @@
   "publishConfig": {
     "access": "public"
   },
-  "engines": {
-    "node": ">=10.12.0",
-    "npm": ">=5.3.0",
-    "yarn": ">=1.9.2"
-  },
```

Comment:
question: are we intentionally removing engine version ranges from our library packages?